src/compton.c

```diff
@@ -2771,9 +2775,8 @@ ev_reparent_notify(session_t *ps, xcb_reparent_notify_event_t *ev) {
         }
         // Otherwise, watch for WM_STATE on it
         else {
-          XSelectInput(ps->dpy, ev->window,
-              determine_evmask(ps, ev->window, WIN_EVMODE_UNKNOWN)
-              | PropertyChangeMask);
+          xcb_change_window_attributes(c, ev->window, XCB_CW_EVENT_MASK, (const uint32_t[]) {
+              determine_evmask(ps, ev->window, WIN_EVMODE_UNKNOWN) | PropertyChangeMask });
```

Comment:
Use `XCB_EVENT_MASK_PROPERTY_CHANGE`?

Reply:
I added a new commit that changes all remaining event masks to XCB defines